### DISCUSSION TITLE
Implement WindowPositionChanged in Labs

### DISCUSSION THREAD

**Discussion by Pdawg-bytes** [CommunityToolkit/Labs-Windows#466] [Ideas]
A conversation in UWP Community took place earlier today, and we discussed getting an event handler up and running for WindowPositionChanged. It’s a useful handler for creating cool visual effects.

@Arlodotexe said that it may be something that could be added to Labs. What do you all think?

**Comment by michael-hawker** [comment]
@Pdawg-bytes assumed you closed this as the other discussion was opened here? https://github.com/CommunityToolkit/Labs-Windows/discussions/468

**Reply by Pdawg-bytes** [reply]
No, I reopened this under a new discussion since the embed was bugged for this one. Check out #467.
